src/caches/abstractbeliefpropagationcache.jl

```diff
@@ -283,3 +283,68 @@ function update(
 )
   return update(Algorithm(alg), bpc; kwargs...)
 end
+
+function rescale_message(bp_cache::AbstractBeliefPropagationCache, partitionedge)
+  return rescale_messages(bp_cache, typeof(partitionedge)[partitionedge])
```

Comment:
```suggestion
  return rescale_messages(bp_cache, [partitionedge])
```
The compiler should be able to figure out the element type, were you seeing issues with that?

Reply:
No I wasn't - will simplify it to let the compiler figure it out